src/Core/Manifest/Module.php

```diff
@@ -288,7 +290,7 @@ public function getRelativeResourcePath($path)
     }
 
     /**
-     * @deprecated 4.0.0:5.0.0 Use ->getResource($path)->getPath() instead
+     * @deprecated 4.0.0 Use ->getResource($path)->getPath() instead
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use getResource($path)->getPath() instead
```
Need to update the version in the notice below

Reply:
Updated